spec/core/SpyStrategySpec.js

```diff
@@ -110,6 +110,62 @@ describe("SpyStrategy", function() {
     })
   });
 
+  describe("#resolveValue", function() {
+    it("allows a resolved promise to be returned", function(done) {
+      if (!jasmineUnderTest.getPromise()) {
+        pending('Environment does not support promises.');
```

Comment:
Try using [`requirePromises`](https://github.com/jasmine/jasmine/blob/110c092c9ece00cd6c882bd573b911c35ff73478/spec/helpers/promises.js) for consistent behavior with other specs.